js/Visua11yNavigationButtonView.js

```diff
@@ -46,13 +46,6 @@ class Visua11yNavigationButtonView extends NavigationButtonView {
   }
 
   setupEventListeners() {
-    const config = Adapt.course.get('_visua11y');
-
-    if (config._location === 'drawer') {
-      this.listenTo(Adapt, 'drawer:closed', this.onDrawerClosed);
-      return;
-    };
-
```

Comment:
```js
    const config = Adapt.course.get('_visua11y');
    if (config._location === 'drawer') return;
```
The return early needs to stay as visua11y can either be displayed in the drawer or a notify. The event listeners for notify should not be attached it visua11y is shown in the drawer.

Reply:
Thanks @oliverfoster - amended in [dc6f9e3](https://github.com/cgkineo/adapt-visua11y/pull/130/commits/dc6f9e3773ef7655f5f5f01b4c0fa526f93d662c)